client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -489,3 +506,36 @@ export const getSiteCountText = ( sites: Array< Site > ) => {
 		comment: '%(siteCount) is no of sites selected, e.g. "2 sites"',
 	} );
 };
+
+export const getBoostRating = ( boostScore: number ) => {
+	if ( boostScore > 90 ) {
+		return 'A';
+	}
+	if ( boostScore > 75 ) {
+		return 'B';
+	}
+	if ( boostScore > 50 ) {
+		return 'C';
+	}
+	if ( boostScore > 35 ) {
+		return 'D';
+	}
+	if ( boostScore > 25 ) {
+		return 'E';
+	}
+	return 'F';
+};
+
```

Comment:
Apologies for not mentioning this before. I took this piece of code from the Jetpack plugin. I think we should refactor this instead of using multiple if statements. Here is one suggestion

```
type BoostRating = 'A' | 'B' | 'C' | 'D' | 'E' | 'F';

interface BoostThreshold {
  threshold: number;
  rating: BoostRating;
}

const BOOST_THRESHOLDS: BoostThreshold[] = [
  { threshold: 90, rating: 'A' },
  { threshold: 75, rating: 'B' },
  { threshold: 50, rating: 'C' },
  { threshold: 35, rating: 'D' },
  { threshold: 25, rating: 'E' },
  { threshold: 0, rating: 'F' },
];

export const getBoostRating = (boostScore: number): BoostRating => {
  for (const { threshold, rating } of BOOST_THRESHOLDS) {
    if (boostScore > threshold) {
      return rating;
    }
  }
  return 'F';
};

```